### PR TITLE
add text styling changes

### PR DESCRIPTION
**Pull request by n8fr8**
This isn't pretty, but makes it possible to set/override the text color using a new "survey_text" color value.

### CODE REVIEW

File: example/src/main/res/layout/main_activity.xml [hunk 1]
```diff
@@ -8,6 +8,8 @@
     <com.quickbirdstudios.surveykit.survey.SurveyView
         android:id="@+id/survey_view"
         android:layout_width="match_parent"
-        android:layout_height="match_parent" />
+        android:layout_height="match_parent"
+        android:background="@color/color_primary_dark"
```
Comment (KlausNie) [inline example/src/main/res/layout/main_activity.xml:12]:
```suggestion
```

File: example/src/main/res/values/colors.xml [hunk 2]
```diff
@@ -15,4 +15,6 @@
     <color name="orange_text">#E75B30</color>
     <color name="orange_normal">#DF552B</color>
     <color name="orange_dark">#AA4020</color>
+
+    <color name="survey_text">#FFFFFF</color>
```
Comment (KlausNie) [inline example/src/main/res/values/colors.xml:19]:
```suggestion
```

File: example/src/main/res/layout/main_activity.xml (hunk 1 above)
Comment (KlausNie) [reply example/src/main/res/layout/main_activity.xml:12]: I'd like to keep the example app as it is for now. The other adjustments are fine!

File: example/src/main/res/values/colors.xml (hunk 2 above)
Comment (KlausNie) [reply example/src/main/res/values/colors.xml:19]: I'd like to keep the example app as it is for now. The other adjustments are fine!

**Comment by coroutineDispatcher** [comment]
Hello. Did you have time to check the reviews and fix them so that we can merge?

**Comment by n8fr8** [comment]
Thanks all. Haven't had time, but appreciate this being resolved!

**Comment by KlausNie** [comment]
Thank you for your PR! I was a necessary and important change and I'm really happy that you took your time and effort to work on it!

I'll try to find a time this week to release a new version👍🏻

**Pull request merged**